### PR TITLE
Fix release generation for all OSs

### PR DESCRIPTION
**Pull request by katcipis**
I'm not sure if other OSes will have /dev/stdout as linux, but it seemed that the outlier is windows =P. So I tried to fix the write stdbin on these other platforms. I also added the release procedure to the CI to avoid this is the future =)

### CODE REVIEW

File: stdbin/write/fd_windows.go [hunk 1]
```diff
@@ -5,9 +5,9 @@ import (
 	"os"
 )
 
-func specialFile(path string) (io.Writer, bool) {
-	if fname == "CON" { // holycrap!
```
Comment (tiago4orion) [inline stdbin/write/fd_windows.go]: o.o
err... this is being used?

File: hack/releaser.sh [hunk 2]
```diff
@@ -48,36 +49,37 @@ for os in $supported_os {
...
-		projectdir <= pwd
-		distar  <= format("%s/dist/nash-%s-%s-%s.tar.gz", $projectdir, $version, $os, $arch)
+		var projectdir <= pwd
+		var distar  <= format("%s/dist/nash-%s-%s-%s.tar.gz", $projectdir, $version, $os, $arch)
 
 		chdir($pkgdir)
-		pkgraw <= ls
-		pkgfiles <= split($pkgraw, "\n")
+		var pkgraw <= ls
+		var pkgfiles <= split($pkgraw, "\n")
```
Comment (tiago4orion) [inline hack/releaser.sh]: lots and lots and lots of var =P

Comment (katcipis) [reply hack/releaser.sh]: yeah =/

File: stdbin/write/fd_windows.go (hunk 1 above)
Comment (katcipis) [reply stdbin/write/fd_windows.go]: I dont know...but the code was there and the build was failing =P. I tried to fix it, but I have nothing against just removing it. Your call =P

**Pull request merged**